agent/containers/images/Dockerfile.base.j2

```diff
@@ -13,9 +13,9 @@ RUN \
     {{ pkgmgr }} module -y disable python38 && \
 {% endif %}
 {% if distro_image.startswith('centos') %}
-    {{ pkgmgr }} install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-{{ distro_image.split(':', 1)[1] }}.noarch.rpm && \
+    {{ pkgmgr }} install -y --setopt=tsflags=nodocs https://dl.fedoraproject.org/pub/epel/epel-release-latest-{{ distro_image.split(':', 1)[1] }}.noarch.rpm && \
```

Comment:
Hey @portante may I know what does this flag signify?

Reply:
`--setopt=tsflags=nodocs`

Reply:
This is taken from Project Atomic "Container Best Practices": http://docs.projectatomic.io/container-best-practices/